src/components/App.js

```diff
@@ -679,7 +700,58 @@ class ReportBuilder {
           tab.items = response.items
           return resolve()
         })
-    }))
+	}))
+  }
+  
+  fetchInventoryList() {
+    return new Promise((resolve, reject) => {
+      let report = this;
+      return GetCharacters(this.cookie, this.account)
+        .then(response => {
+          if (response.status === 404) {
+            this.notice(
+              `No tabs found in ${league} league. Try another one? 🤷`
+            );
+            this.notice(response);
+          }
+
+          if (response.status === 403) {
+            events.emit("clear_config");
+            this.notice(`Session expired. You have been logged out.`);
+          }
+
+          if (!response.data) {
+            this.notice(
+              `No tabs found in ${league} league. Try another one? 🤷`
+            );
+          } else if (response.status === 200) {
+            return response;
+          }
+
+          throw null;
+        })
+        .then(response => {
+          if (response.status === 200) {
+            let inventoryTabs = [];
+            response.data.forEach(element => {
+              if (element.league === report.settings.league) {
+                let tab = JSON.parse(
```

Comment:
Can't let this in due to the JSON.parse of a random object here, could easily just be an empty object it seems since you override everything below.

Reply:
fixed it